academy/0.0-B9lab-Blockchains/3_managed.md

```diff
@@ -0,0 +1,134 @@
+
+<HighlightBox type="info">
+
+**Ether** is the **native cryptocurrency** of the Ethereum platform, as Bitcoin is the native cryptocurrency of the Bitcoin blockchain.
+
+</HighlightBox>
+
+The protocol defines the **Gas cost** for each operation in the EVM. The halting problem is resolved by attaching financial costs to every step in a program and by limiting the funds available. Transactions that do not compute successfully and are not on budget are rejected.
+
+In Ethereum, **smart contracts** are stored on every single node - they are public. In practice, a smart contract in the EVM is an autonomous agent with an internal account. The most popular language for writing such a contract is **Solidity**.
+
+<ExpansionPanel title="Faster block time">
+
+Ethereum has a much faster block time than Bitcoin - currently it is around 15 seconds. This is made possible thanks to Ethereum's implementation of the [GHOST protocol](http://www.cs.huji.ac.il/~yoni_sompo/pubs/15/inclusive_full.pdf). Ethereum implements a variation of Bitcoin's PoW consensus algorithm called Ethash, which is intended to be [ASIC-resistant and GPU friendly](https://github.com/ethereum/wiki/wiki/Ethash-Design-Rationale). Core developers of the Ethereum project plan to switch to the **proof-of-stake (PoS)** consensus algorithm in the future.
+
+A well-understood challenge related to reduced block time stems from network latency. Since nodes do not learn about newly discovered blocks simultaneously, at any given time a portion of the miners will be working on already solved (i.e. old) blocks. If these miners find a solution, they might not be rewarded and a "fork" of the chain results, in which rival chains with a common true history spread through the network. The rate at which such transient forks occur increases predictably with shorter block times. While Bitcoin addresses this efficiency issue with a relatively long block time of 10 minutes, Ethereum addresses this concern with a partial reward strategy: valid blocks that aren't ultimately included in the canonical chain (because another chain becomes longer) are still included on the side and known as **uncles**. Miners of uncles receive a smaller reward than regular block miners. 
+
+![Uncles in Ethereum](images/00_18_block_uncles-01.png)
+
+The precise mechanism employed by Ethereum is called GHOST. GHOST includes so-called uncles that are propagated into the network too late to rise to the level of network consensus. This increases the total difficulty of the chain, makes smaller block times possible, and rewards miners of uncles for contributing to the overall strength of the network.
+
+Ethereum's solution to achieve faster block times draws attention to the universal challenge of blockchain technology, performance, and throughput. Cosmos solves for performance in two ways: with a faster consensus process, and by introducing parallelism.
```

Comment:
@xavierlepretre , I query the sentence **Ethereum's solution to achieve faster block times draws attention to the universal challenge of blockchain technology, performance, and throughput.**

It is ambiguous whether there are:

1.  **three** challenges (blockchain technology, performance, and throughput); or 
2. **two** challenges _of blockchain_ (performance, and throughput).

Reply:
It is 2 challenges.